xtext-gradle-builder/src/main/java/org/xtext/gradle/builder/XtextGradleBuilder.xtend

```diff
@@ -230,12 +230,16 @@ class XtextGradleBuilder implements IncrementalXtextBuilder {
 			attachToEmfObject(resourceSet)
 			language2GeneratorConfig.putAll(
 				gradleRequest.generatorConfigsByLanguage.mapValues [ gradleConfig |
+					val javaVersion = JavaVersion.fromQualifier(gradleConfig.javaSourceLevel.toString)

```

Comment:
am not sure how we should react here.
Java 11 as default is not available as we compile against old version.
maybe we should stop here and report error instead?